snapcraft.yaml

```diff
@@ -60,3 +65,35 @@ parts:
     build-packages:
       - python3-pip
       - python3-setuptools
+      - libmagic-dev
+      - pkg-config
+      - libzip-dev
+      - liblz4-dev
+      - libssl-dev
+      - libbsd-dev
+      - libcapstone-dev
+      - cmake
+      - libuv1-dev
+    stage-packages:
+      - libuv1
+    after: [ninja, acr]
+  ninja:
```

Comment:
I don't know anything about snap... but why are you installing both ninja and acr?

Reply:
Both are remnants from an older version of the `snapcraft.yaml`, circa two years ago.
I removed them.